app/routes/account.$accountId.$appId/hooks/useSubscriptionSync.tsx

```diff
@@ -102,7 +101,7 @@ const useSubscriptionSync = ({
       searchParams.delete("cancelError")
       setShowErrorModel(true)
     }
-  }, [searchParams, endpoint, updatePlanFetcher, setSearchParams])
+  }, [searchParams, endpoint, updatePlanFetcher, setSearchParams, setShowErrorModel])
```

Comment:
Just a general comment, setState functions can be ignored and not passed to dependency array. React ensures they're not going to change.

Reply:
This was called out by the linter and our current rules.